cvmfs/catalog_mgr_rw.cc

```diff
@@ -12,9 +12,12 @@
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
+#include <ctime>
```

Comment:
To be more precise: if you need the include for the patch, it is of course part of the patch pull request.  But if the include was missing in the first place, this should be a separate pull request with a fix.